drive-characterization/README.md

```diff
@@ -73,10 +73,17 @@ Preparation: make sure the code won't make your robot go crazy
 5. Open SmartDashboard/Shuffleboard/OutlineViewer and ensure that the
    l_encoder and r_encoder values are incrementing positively the correct
    distance in **feet** when you push the robot forward
+6. If you're planning to use angular mode (see below), spin your robot
+   clockwise and ensure that the gyro_angle is increasing positively.
+   Spin your robot > 360 degrees and ensure that the gyro_angle does not
+   wrap back around to 0. Also ensure that the gyro_angle is in radians.
 
 Now you're ready to characterize your robot! On your data gathering computer,
 launch data_logger.py (you can double-click it on Windows). Enter in your
-team number or robot IP address when prompted.
+team number or robot IP address when prompted. You will then be prompted to
+enter "linear" or "angular" mode. You should enter "linear" if you don't know
+which to choose. *Note:* If you select angular mode you must manually reverse
```

Comment:
Why not say "your robot must drive correctly?", and let the user figure out the rest, given it varies based on wiring/etc.

Reply:
How does this sound:

*Note:* Your motor directions must be set differently for angular and linear mode. For linear mode the robot should drive forwards when you apply the same percent output to all motors. For angular mode your robot should turn in place when the same percent output is applied to all motors.

Reply:
That sounds reasonable.

Reply:
Wait a second, so you're asking a team to reconfigure their physical robot when in a particular mode? That seems very unreasonable. Why not just make the program command the robot one way or another?

Reply:
The reason I didn’t do that was because of a conversation with on the FRC Discord with @Oblarg. He felt that for an advanced feature like this it was easier to just tell users to do it themselves, instead of changing the autospeed NT value to be a number array.

I think that doing it automatically is cleaner, but it also breaks past user programs, which isn’t ideal. What’s your opinion?